src/gql/apollo.js

```diff
@@ -35,17 +37,50 @@ const makeWebSocketLink = () => {
 }
 
 const createApolloClient = () => {
-  const link = split(
-    ({ query }) => {
-      const definition = getMainDefinition(query)
-      return (
-        definition.kind === "OperationDefinition" &&
-        definition.operation === "subscription"
-      )
-    },
-    makeWebSocketLink(),
-    makeHttpLink()
-  )
+  const link = ApolloLink.from([
+    // suspending errors, preventing to fire them
+    new ApolloLink((operation, forward) => {
+      return new LinkObservable(observer => {
+        let sub
+        sub = forward(operation).subscribe({
+          next: result => {
+            // check if we have errors
+            if (!result.errors) {
+              observer.next(result)
+            } else {
+              // pass errors to sentry
+              result.errors.map(err => {
+                Sentry.captureException(err, {
+                  level: "error"
```

Comment:
why do we need to set the level?

Reply:
Don't know for sure, just used as I have found in the example

Reply:
we don't use this anywhere else, does it provide any additional value? if not let's remove it to stay coherent in the codebase.

Reply:
Updated